docs/examples/algo/complete-example.py

```diff
@@ -1,87 +1,106 @@
 import csv
 import datetime
-
 import numpy
+import os
+import yaml
```

Comment:
In all 3 examples we are using a single YAML file with all the params. This YAML is heavily commented to explain each param. Using YAML vs native python module makes it a bit more obvious in the docs that it's a configuration file. Another (small) benefit is that it displays in a more compact manner in the docs and it's easier to read than the native python module.